src/posts/2021-01-12-k8s-pocast/index.md

```diff
@@ -0,0 +1,13 @@
+---
+date: '2021-01-12T17:00:00.000Z'
+title: 'Cilium with Thomas Graf on Google Kubernetes Podcast'
+ogImageUrl: 'https://pbs.twimg.com/profile_images/1095059886730137600/icMSa8g4_400x400.jpg'
```

Comment:
Replace with this image:
https://pbs.twimg.com/profile_images/1095059886730137600/icMSa8g4.jpg